src/components/authorization/policies/by-role/regional-director.policy.ts

```diff
@@ -6,7 +6,7 @@ import { member, Policy, Role, sensMediumOrLower } from '../util';
 
   r.Partnership.read,
   r.Project.when(member).edit.specifically(
-    (p) => p.rootDirectory.when(sensMediumOrLower).read,
+    (p) => p.rootDirectory.edit.when(sensMediumOrLower).read,
```

Comment:
So here the root directory was not editable, but now you are making it editable?

Reply:
Logically there's no change here. This was added to restore previous functionality with this bug fix.

We want `rootDirectory` to be editable when you are a member.
Which the `.edit` does since it takes the closest previously defined condition.

Like above, we are additionally adding read access for low/med sens projects even if you are not a member.
But because we called that out we also need to restore the object-level permission on this property.